pkg/services/accesscontrol/resourcepermissions/store_bench_test.go

```diff
@@ -137,7 +140,10 @@ func generateTeamsAndUsers(b *testing.B, db *sqlstore.SQLStore, users int) ([]in
 	teamSvc := teamimpl.ProvideService(db, db.Cfg)
 	numberOfTeams := int(math.Ceil(float64(users) / UsersPerTeam))
 	globalUserId := 0
-
+	orgSvc, err := orgimpl.ProvideService(db, db.Cfg, quotatest.New(false, nil))
+	require.NoError(b, err)
+	usrSvc, err := userimpl.ProvideService(db, orgSvc, db.Cfg, nil, nil, &quotatest.FakeQuotaService{})
```

Comment:
I didn't bother because it didn't make a difference in the test results, and the files themselves were already similarly inconsistent, but I can go and clean up the changes i made. I'm won't fix all the other inconsistencies - the ones I didn't cause - in this pr though.